src/transmissionSettings/sideEffects.ts

```diff
@@ -0,0 +1,14 @@
+import { reduxActionSideEffect } from 'redux-easy-mode'
+
+import apiInstance from '@src/api/apiInstance'
+
+import actions from './actions'
+
+reduxActionSideEffect(actions.update, (action, dispatch) => {
+  async function run() {
+    await apiInstance.callServer('session-set', action.payload)
+    await dispatch(actions.get())
+  }
+
+  run()
```

Comment:
I might move this to the action itself, rather than a side effect. Kinda six of one, half a dozen of the other.